QUARKUS-5623.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-5623 - Logging in OpenTelemetry
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5623
+
+Upstream PRs:
+    - https://github.com/quarkusio/quarkus/pull/38239
+    - https://github.com/quarkusio/quarkus/pull/46354
+
+Documentation: https://quarkus.io/version/main/guides/opentelemetry-logging
+
+## Preview status
+OpenTelemetry logging is currently considered dev-preview in quarkus.
+
+## Scope of testing
+Test OpenTelemetry quarkus extension.
+Verify that quarkus application is sending its logs to the Grafana backend.
+Verify that log messages as well as log levels are correct.
+
+## Getting familiar with the feature
+Following actions were taken to ensure familiarity:
+- Focus on exploratory testing of the feature
+- Ensure good user experience and simplicity of use
+- Check the community status of the extension/feature, preview and experimental ones should be considered carefully
+
+## Automated test development
+- Test suite - New scenarios for baremetal and OCP execution.
```

Comment:
`OCP execution` ... so the TF Grafana service will support both baremetal and OCP environments?

Reply:
Yes, I'm already working on this in the FW and it works fine in both envs - https://github.com/quarkus-qe/quarkus-test-framework/pull/1503